packages/venia-ui/lib/components/CartPage/PriceSummary/queries/GiftCardSummary.ce.js

```diff
@@ -0,0 +1,7 @@
+import gql from 'graphql-tag';
+
+export const GiftCardSummaryFragment = gql`
+    fragment GiftCardSummaryFragment on Cart {
+        __typename
```

Comment:
Interesting -- I was hoping to eventually avoid having to maintain a "no-op" fragment. Is it possible for the plugin to replace it with an empty fragment instead?

Reply:
The plugin has nothing to do with Queries or GQL in general. It resolves files based on env variables. If Apollo allows empty strings you make this export an empty string.